account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/AuthenticateHandler.java

```diff
@@ -69,7 +69,7 @@ public APIGatewayProxyResponseEvent handleRequest(
                                 }
                                 LOGGER.info(
                                         "User has successfully Logged in. Generating successful AuthenticateResponse");
-                                return generateApiGatewayProxyResponse(200, "");
+                                return generateApiGatewayProxyResponse(204, "");
```

Comment:
Could we abstract this into a new method, e.g `generateEmptySuccessApiGatewayResponse()` rather than repeat this line?

Reply:
I have updated 👍